README.md

```diff
@@ -26,15 +26,17 @@ Uses SponsorBlock data licensed used under CC BY-NC-SA 4.0. More details: https:
     ```
 
 3. Install python project dependencies 
-    - It is recomended to create a virtual environment instead of installing libraries globally to avoid conflicts with any other python projects. If you like, you can start a new virtual environment by:
+    - It is *recomended to create a virtual environment* instead of installing libraries globally to avoid conflicts with any other python projects. 
+    - Virtual environments can usually live in the project root directory. If you like, you can start a new virtual environment by:
         ```bash
-        python -m venv venv/
-        source venv/bin/activate (if you are using linux shell)
-        venv\Scripts\activate (if using cmd or powershell)
+        cd <your project root directory>
+        python -m venv <venv name>/
+        cd <venv name>\Scripts\activate (On cmd or powershell)
+        source <venv name>/bin/activate (On linux shell)
         ```
     - Install python project dependencies using pip
         ```bash
-        python -m pip install -r requirements.txt
+        python pip install -r requirements.txt
```

Comment:
its pip dont know how python gpt there....

Reply:
Does that work for you?

Reply:
python -m pip install -r requirements.txt 
or 
pip install -r requirements.txt works. Or depending the system python3 or pip3

I choose python -m pip install -r requirements.txt because it doesn't matter if pip is not on PATH as long as python is on PATH

Reply:
I think it's just `pip install -r requirements.txt ` which we had originally.

Reply:
you had -m previously

Reply:
Does python -m pip install -r requirements.txt not work? It's safer since you know it will call the same python installation constantly throughout the setup. This is helpful if someone is installing the bot but has multiple python versions on the computer.

Reply:
Hmm, my bad. It looks like the Python documentation recommends `python -m pip install`. Let's just keep it as it is right now. 

https://docs.python.org/3/tutorial/venv.html

Reply:
> you had -m previously
> 
> 
> 
> 

But the change you proposed is python pip install -r requirements.txt